src/config.ini

```diff
@@ -39,14 +39,16 @@ rebuild = build --no-cache
 
 [mysql-backup-filter]
 
-typo3[] = "/^cachingframework_.*/i"
-typo3[] = "/^cf_.*/i"
-typo3[] = "/^cache_.*/i"
-typo3[] = "/^index_.*/i"
-typo3[] = "/^sys_log$/i"
-typo3[] = "/^sys_history$/i"
-typo3[] = "/^tx_extbase_cache.*/i"
-typo3[] = "/^tx_extensionmanager_.*/i"
+all[] = "/^.*$/i"
+
+typo3caching[] = "/^cachingframework_.*/i"
```

Comment:
But then doesn't all clisync.yml files in the project has to be changed? There is a filter setting which is currently set to `typo3` instead of `typo3caching`